Content.IntegrationTests/Tests/EntityTest.cs

```diff
@@ -75,6 +76,12 @@ public async Task Test()
                 });
 
             await server.WaitIdleAsync();
+
+            server.Stop();
```

Comment:
This is way too boilerplatey and error prone to have to stick at the end of every integration test. I actually tried to make it so this was done automatically with a `[TearDown]` so I would prefer if you figured out why this isn't working instead.